skimage/measure/_boundary.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+"""
+Code copied from https://github.com/machine-shop/deepwings/blob/master/deepwings/method_features_extraction/image_processing.py#L185
+"""
+
+
+def _moore_neighborhood(current, backtrack):  # y, x
+    """Returns clockwise list of pixels from the moore neighborhood of current\
+    pixel:
+    The first element is the coordinates of the backtrack pixel.
+    The following elements are the coordinates of the neighboring pixels in
+    clockwise order.
+    Parameters
+    ----------
+    current ([y, x]): Coordinates of the current pixel
+    backtrack ([y, x]): Coordinates of the backtrack pixel
+    Returns
+    -------
+    List of coordinates of the moore neighborood pixels, or 0 if the backtrack
+    pixel is not a current pixel neighbor
+    """
+
+    operations = np.array(
+        [[-1, 0], [-1, 1], [0, 1], [1, 1], [1, 0], [1, -1], [0, -1], [-1, -1]]
+    )
+    neighbors = (current + operations).astype(int)
+
+    for i, point in enumerate(neighbors):
+        if np.all(point == backtrack):
+            # we return the sorted neighborhood
+            return np.concatenate((neighbors[i:], neighbors[:i]))
+    else:
+        raise RuntimeError("The backtrack is not on the neighborhood")
+
+
+def trace_boundary(coords):
+    """Coordinates of the region's boundary. The region must not have isolated
+    points.
+    Parameters
+    ----------
+    coords : obj
+        Obtained with skimage.measure.regionprops()
```

Comment:
```suggestion
        Obtained with skimage.measure.regionprops()
        
```

Reply:
I tried adding a newline here because there is a formatting problem which makes the Python36-x64 build fail on the Azure CI